plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -960,8 +961,26 @@ def _onPrintTimeMaterialEstimates(self, message: Arcus.PythonMessage) -> None:
         """
 
         material_amounts = []
+        no_use_warnings = []
         for index in range(message.repeatedMessageCount("materialEstimates")):
-            material_amounts.append(message.getRepeatedMessage("materialEstimates", index).material_amount)
+            material_use_for_tool = message.getRepeatedMessage("materialEstimates", index).material_amount
+            if isnan(material_use_for_tool):
+                material_amounts.append(0.0)
+                if self._global_container_stack.extruderList[int(index)].isEnabled:
+                    no_use_warnings.append(index)
+            else:
+                material_amounts.append(material_use_for_tool)
+
+        if no_use_warnings:
+            extruder_names = [self._global_container_stack.extruderList[int(idx)].definition.getName() for idx in no_use_warnings]
+            Message(
+                text=catalog.i18nc("@message", "At least one extruder that wasn't disabled, remains unused in this print (preview). "
+                                               f"Unused extruders: '<b>{", ".join(extruder_names)}</b>'. This can sometimes become a problem "
+                                               "(for example when the bed-temperature is adjusted by the material-profile present in the unused extruder). "
+                                               "It therefore might be desirable to disable these unused extruders manually, depending on the situation. "),
```

Comment:
[nitpick] may I suggest slightly altered version? I believe that this is a bit more clear about the potential impact on quality and reliability

```suggestion
                text=catalog.i18nc("@message", ""**Warning**: not all extruders that were left enabled are actually extruding in this printjob."  
f"Unused extruders: '<b>{", ".join(extruder_names)}</b>'." 
"This may lead to sub-optimal settings! To enjoy optimal printed part quality and reliability, please consider to disable the unused extruder(s)!"),
```

Reply:
Thanks! I think that might indeed be more clear -- excepting the bit about 'sub-optimal settings', the sort of users that need this warning are just going to ignore that I feel, so a concrete example might help.

What do you think @HellAholic ?

Reply:
I'm ok with making the wording more firm to convey the message better. The last sentence feels a bit too aggressive though. I would suggest to involve PM or UX to decide on the final wording.